diff -up meson-1.0.1/data/macros.meson.3~ meson-1.0.1/data/macros.meson
--- meson-1.0.1/data/macros.meson.3~	2023-04-05 20:53:30.982710784 +0200
+++ meson-1.0.1/data/macros.meson	2023-04-05 20:54:12.181060739 +0200
@@ -8,8 +8,14 @@
     %set_build_flags \
     export LANG="en_US.UTF-8" \
     export LC_ALL="en_US.UTF-8" \
+%ifnarch noarch \
+%if %{cross_compiling} \
+    export CROSSCOMPILE_FILE="--cross-file=%{_datadir}/meson/toolchains/%{_target_platform}%{?prefer_gcc:-gcc}.cross" \
+%endif \
+%endif \
     %{shrink:%{__meson} setup \
         --buildtype=plain \
+	$CROSSCOMPILE_FILE \
         --prefix=%{_prefix} \
         --libdir=%{_libdir} \
         --libexecdir=%{_libexecdir} \
