diff -Naur meson-0.50.0/data/macros.meson meson-0.50.0.tpg/data/macros.meson
--- meson-0.50.0/data/macros.meson	2019-02-07 09:08:55.000000000 +0000
+++ meson-0.50.0.tpg/data/macros.meson	2019-03-11 14:58:44.031000000 +0000
@@ -1,9 +1,13 @@
 %__meson %{_bindir}/meson
 %__meson_wrap_mode nodownload
 %__meson_auto_features enabled
+%_vpath_srcdir .
+%_vpath_builddir build
 
 %meson \
     %set_build_flags \
+    export LANG="en_US.UTF-8" \
+    export LC_ALL="en_US.UTF-8" \
     %{shrink:%{__meson} \
         --buildtype=plain \
         --prefix=%{_prefix} \
@@ -25,10 +29,13 @@
 	%{nil}}
 
 %meson_build \
+    export LC_ALL="en_US.UTF-8" \
     %ninja_build -C %{_vpath_builddir}
 
 %meson_install \
+    export LC_ALL="en_US.UTF-8" \
     %ninja_install -C %{_vpath_builddir}
 
 %meson_test \
+    export LC_ALL="en_US.UTF-8" \
     %ninja_test -C %{_vpath_builddir}
