diff -uraN meson-0.55.0/data/macros.meson omv-meson-0.55.0/data/macros.meson
--- meson-0.55.0/data/macros.meson	2020-07-12 16:25:51.000000000 +0200
+++ omv-meson-0.55.0/data/macros.meson	2020-07-13 21:35:25.079211019 +0200
@@ -24,6 +24,30 @@
         %{_vpath_srcdir} %{_vpath_builddir} \
         %{nil}}
 
+%meson32 \
+        %set_build_flags \
+        export LANG="en_US.UTF-8" \
+        export LC_ALL="en_US.UTF-8" \
+        PKG_CONFIG_LIBDIR="%{_prefix}/lib/pkgconfig:%{_datadir}/pkgconfig:$PKG_CONFIG32_LIBDIR" PKG_CONFIG_PATH="%{_prefix}/lib/pkgconfig:%{_datadir}/pkgconfig:$PKG_CONFIG32_PATH" CC=gcc CXX=g++ CFLAGS="`echo $CFLAGS |sed -e 's, -m64,,g;s, -mx32,,g;s, -flto,,g'` -m32" CXXFLAGS="`echo $CXXFLAGS |sed -e 's, -m64,,g;s, -mx32,,g;s, -flto,,g'` -m32" LDFLAGS="`echo $LDFLAGS |sed -e 's, -m64,,g;s, -mx32,,g;s, -flto,,g'` -m32" %{shrink:%{__meson} \
+            --buildtype=plain \
+            --prefix=%{_prefix} \
+            --libdir=%{_prefix}/lib \
+            --libexecdir=%{_libexecdir} \
+            --bindir=%{_bindir} \
+            --sbindir=%{_sbindir} \
+            --includedir=%{_includedir} \
+            --datadir=%{_datadir} \
+            --mandir=%{_mandir} \
+            --infodir=%{_infodir} \
+            --localedir=%{_datadir}/locale \
+            --sysconfdir=%{_sysconfdir} \
+            --localstatedir=%{_localstatedir} \
+            --sharedstatedir=%{_sharedstatedir} \
+            --wrap-mode=%{__meson_wrap_mode} \
+            --auto-features=%{__meson_auto_features} \
+            %{_vpath_srcdir} %{_vpath_builddir}32 \
+    	%{nil}}
+
 %meson_build \
     %{shrink:%{__meson} compile \
         -C %{_vpath_builddir} \
