diff -up meson-1.4.0/data/macros.meson.1~ meson-1.4.0/data/macros.meson
--- meson-1.4.0/data/macros.meson.1~	2024-03-12 18:57:31.000000000 +0100
+++ meson-1.4.0/data/macros.meson	2024-03-12 23:43:43.781364641 +0100
@@ -2,9 +2,13 @@
 %__meson_wrap_mode nodownload
 %__meson_auto_features enabled
 %__meson_verbose 1
+%_vpath_srcdir .
+%_vpath_builddir build
 
 %meson \
     %set_build_flags \
+    export LANG="en_US.UTF-8" \
+    export LC_ALL="en_US.UTF-8" \
     %{shrink:%{__meson} setup \
         --buildtype=plain \
         --prefix=%{_prefix} \
@@ -26,6 +30,7 @@
         %{nil}}
 
 %meson_build \
+    export LC_ALL="en_US.UTF-8" \
     %{shrink:%{__meson} compile \
         -C %{_vpath_builddir} \
         -j %{_smp_build_ncpus} \
@@ -33,6 +38,7 @@
         %{nil}}
 
 %meson_install \
+    export LC_ALL="en_US.UTF-8" \
     %{shrink:DESTDIR=%{buildroot} %{__meson} install \
         -C %{_vpath_builddir} \
         --no-rebuild \
